tests/unit/models/test_build.py

```diff
@@ -77,17 +77,20 @@ def test_builds_comparison_other_types(self):
 
     def test_build_str(self):
         """Test Build __str__ method."""
-        self.assertEqual(str(self.build), f'Build: {self.build_id}')
+        self.assertIn('Build: ', str(self.build))
```

Comment:
The tests modified in this change are less restrictive than before. Using `assertIn`makes sense, but I would also test for the particulars of each model. For example in this case, I would add `self.assertIn(build_id, str(self.build))` to make sure that the __str__ method of the models grabs the right attributes, and not just the header.

Reply:
That makes sense. 
I will make them a little more restrictive.